static/bidder-params/admixer.json

```diff
@@ -8,7 +8,7 @@
     "zone": {
       "type": "string",
       "description": "Zone ID.",
-      "pattern": "^([a-fA-F\\d\\-]{36})$"
+      "pattern": "^([a-fA-F\\d\\-]{32,36})$"
```

Comment:
Can you please add tests for this change as well in `adapters/admixer/params_test.go`?

Reply:
Hi @mansinahar , sure, we will add tests.